### PR TITLE
fix: add litix to csp

### PR DESCRIPTION
**Pull request by tomwisecodes**
## Description

Was seeing a lot of errors for the home page video from mux 

image

### CODE REVIEW

File: apps/nextjs/src/middlewares/csp.ts [hunk 1]
```diff
@@ -56,21 +56,25 @@ const mux = {
     "https://mux.com",
     "https://*.mux.com",
     "https://stream.mux.com",
+    "https://*.litix.*",
```
[the remark this answers is not in the document]
Comment (tomwisecodes) [reply apps/nextjs/src/middlewares/csp.ts:59]: Hasnt worked anyway

[the remark this answers is not in the document]
Comment (codeincontext) [reply apps/nextjs/src/middlewares/csp.ts]: Agree with this!

**Review by codeincontext — APPROVED** [review]

**Comment by johnrobeds** [comment]
All the mux domains can be first party via `*.video.thenational.academy` and `mux-data`, `mux-litix`, `mux-thumbnail-image` so we should serve the video first party, and then the CSP can stay with the `*.thenational.academy` though the nested `*.video.` subdomain might be need to be added.

**Pull request merged**